OpenSim/Common/Storage.cpp

```diff
@@ -3144,14 +3213,15 @@ bool Storage::parseHeaders(std::ifstream& aStream, int& rNumRows, int& rNumColum
         }
         firstLine=false;
     }
+
+    if (_fileVersion < 1) {
```

Comment:
What's the rationale for using `1` instead of `LatestVersion`?

Reply:
Latest version is 2. Only 1 or earlier goes through the conventional pathway (not through the `FileAdpater`).